### DISCUSSION TITLE
At least two owners approve before they can be merged PR.

### DISCUSSION THREAD

**Discussion by LiangliangSui** [apache/fory#1473] [General]
Hi all,

Currently, we can merge PR as long as there is an owner approve. In order to better solicit opinions on merging PR, can we consider setting an owner approve limit of 2?

**Comment by chaokunyang** [comment]
@LiangliangSui That's what I want to do too. But our community is still in early stage, not all code owner are always online, and unfortunately
 we don't have enough owners at this time. So I suggest to defer this to next month, hopefully we can have a stronger community.

**Comment by PragmaTwice** [comment]
Yeah, as said by @chaokunyang , currently the number of community members is too small to sustain the 2-approval mechanism.
